src/app/Fake.DotNet.Testing.SpecFlow/SpecFlow.fs

```diff
@@ -59,16 +59,16 @@ let run setParams =
 
     let commandLineBuilder = 
         new StringBuilder()
-        |> StringBuilder.append           parameters.SubCommand
```

Comment:
Can we apply a similar change as the other PR (for both `SpecFlow` and `SpecFlowNext`?

Should we add a `Obsolete` attribute?

Reply:
I think for SpecFlowNext we should use the Args.toWindowsCommandLine too. For the old one I wouldn't do this, as this would change behavior maybe.
Obsolete attribute is ok I think. I will put this in there

Reply:
I think if it changes behavior it was a bug before ...

Reply:
So be it ;) I'll change it too then